stdlib/_collections_abc.pyi

```diff
@@ -79,3 +83,8 @@ class dict_items(ItemsView[_KT_co, _VT_co], Generic[_KT_co, _VT_co]):  # undocum
     if sys.version_info >= (3, 10):
         @property
         def mapping(self) -> MappingProxyType[_KT_co, _VT_co]: ...
+
+if sys.version_info >= (3, 12):
+    class Buffer(Protocol):
```

Comment:
Oh wait, this should be `@runtime_checkable` as well, right? (Sorry for the piecemeal review!!)

```suggestion
    @runtime_checkable
    class Buffer(Protocol):
```